### PR TITLE
{bio}[foss/2018a] shovill v0.9.0, KMC v3.1.0, Lighter v1.1.1, Pilon 1.22, SKESA 2.9.1 

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~#6543~~ (BWA), ~~#6542~~ (Mash), ~~#6544~~ (pigz), ~~#6545~~ (SPAdes), ~~#6546~~ (MEGAHIT), ~~#6547~~ (FLASH), ~~#6548~~ (ncbi-vdb), ~~#6549~~ (seqtk), ~~#6550~~ (Velvet)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2004.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ad9d9cb3610089ee344b579b6189de27 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 19 out of 19 (5 easyconfigs in this PR)
node2608.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a4de979de8b53dbb9b7844bde843f9db for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 16 out of 18 (5 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.5.1804, Intel Xeon E312xx (Sandy Bridge), Python 2.7.5
See https://gist.github.com/3ab8fc25b65d2e6004cbc39d06f98a9e for a full test report.

**Comment by boegel** [comment]
@verdurin That build failure look quite suspicious... `error: cannot find -lm`

Was something (temporarily) broken on that system? Can you try again with `eb --from-pr 6553 KMC-3.1.0-foss-2018a.eb`?

**Comment by verdurin** [comment]
@boegel It is suspicious. However, it fails the same way.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 8 out of 8 (5 easyconfigs in this PR)
cermis - Linux debian 9.4, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/ba31a431876fcba2bf0641aa799c90c4 for a full test report.

**Comment by vanzod** [comment]
@verdurin Any update on your issue? I would like to get this merged ASAP.

**Comment by boegel** [comment]
@verdurin Taking another look at this, it looks like the problem may be that you don't have `glibc-static` installed, since this is what fails:

```
$ g++ -lm -static ...  -o bin/kmc_dump ...
/home/centos/.local/easybuild/software/binutils/2.28-GCCcore-6.4.0/bin/ld.gold: error: cannot find -lc
/home/centos/.local/easybuild/software/binutils/2.28-GCCcore-6.4.0/bin/ld.gold: error: cannot find -lpthread
/home/centos/.local/easybuild/software/binutils/2.28-GCCcore-6.4.0/bin/ld.gold: error: cannot find -lm
```

So, if `libm.a` isn't there, that would explain the failure. On my system:

```
$ rpm -qf /usr/lib64/libm.a
glibc-static-2.17-222.el7.x86_64
```

**Comment by verdurin** [comment]
@boegel yes, can confirm I don't have `glibc-static` installed, so that should go in `osdependencies`?

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (5 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.5.1804, Intel Xeon E312xx (Sandy Bridge), Python 2.7.5
See https://gist.github.com/74882786968ea22f948c8f326c3c4a2d for a full test report.

**Comment by verdurin** [comment]
Test succeeded after installing `glibc-static`.

**Comment by boegel** [comment]
@verdurin Thanks for the confirmation, I think adding `glibc-static` as an OS dep reasonable for now.

A better option would be to prevent static linking hardcoded I guess, but that can be picked up on in a future PR if needed...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
gligar01.gligar.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/bcd368e5ad7d43779bd8be9ddae8f496 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (5 easyconfigs in this PR)
node2400.golett.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/4fb686eed1226a9804cceb02c93059ac for a full test report.

File: easybuild/easyconfigs/s/SKESA/SKESA-2.9.1-foss-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+name = 'SKESA'
+version = '2.9.1'
+
+homepage = 'https://github.com/ncbi/ngs-tools/tree/master/tools/skesa'
+description = "SKESA is a de-novo sequence read assembler for cultured single isolate genomes based on DeBruijn graphs."
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = ['https://github.com/ncbi/ngs-tools/archive/']
```
Comment (boegel) [inline easybuild/easyconfigs/s/SKESA/SKESA-2.9.1-foss-2018a.eb]: Download location + version was correct in #7275

**Pull request merged**